app/javascript/react/api/apiEndpoints.ts

```diff
@@ -1,7 +1,9 @@
 interface ApiEndpoints {
   readonly fetchFixedStreamById: (id: number) => string;
+  readonly fetchPartoFMovingStream: (id: number, startDate: string, endDate: string) => string;
 }
 
 export const API_ENDPOINTS: ApiEndpoints = {
   fetchFixedStreamById: (id) => `/fixed_streams/${id}`,
+  fetchPartoFMovingStream: (id, startDate, endDate) => `/stream_daily_averages/${id}?start_date=${startDate}&end_date=${endDate}`,
```

Comment:
Could we come up with a more descriptive name for it? `fetchSelectedDateRangeStream` or `fetchDateRangeStreamById` ?